api/scripts/bigint/answer/copy-newly-inserted-rows-to-answerId-tmp-bigint-tables.js

```diff
@@ -0,0 +1,112 @@
+           WHERE bms."tableName" = 'knowledge-elements'`
+      )
+      .transacting(trx);
+
+    await knex
+      .raw(
+        `UPDATE "bigint-migration-settings" bms
+           SET "endsAtId" = (SELECT MAX(id) FROM "answers")
+           WHERE bms."tableName" = 'answers'`
+      )
+      .transacting(trx);
+
+    await knex
+      .raw(
+        `UPDATE "bigint-migration-settings" bms
+           SET "endsAtId" = (SELECT MAX(id) FROM "knowledge-elements")
+           WHERE bms."tableName" = 'knowledge-elements'`
+      )
+      .transacting(trx);
+
+    await knex
+      .raw(
+        `CREATE OR REPLACE FUNCTION "insert_answers_in_answers_bigint"()
+                    RETURNS TRIGGER AS
+                    $$
+                    BEGIN
+                      INSERT INTO "answers_bigint" ("id", "value", "result", "assessmentId", "challengeId", timeout, "resultDetails", "timeSpent", "isFocusedOut")
+                      VALUES         (NEW."id"::BIGINT, NEW."value", NEW."result", NEW."assessmentId", NEW."challengeId", NEW."timeout", NEW."resultDetails", NEW."timeSpent", NEW."isFocusedOut");
+                     RETURN NEW;
+                    END
+                    $$ LANGUAGE plpgsql;`
+      )
+      .transacting(trx);
+
+    await knex
+      .raw(
+        `CREATE TRIGGER "trg_answers"
+                    AFTER INSERT ON "answers"
+                    FOR EACH ROW
+                    EXECUTE FUNCTION "insert_answers_in_answers_bigint"();`
+      )
+      .transacting(trx);
+
+    await knex
+      .raw(
+        `CREATE OR REPLACE FUNCTION "insert_knowledge-elements_in_knowledge-elements_bigint"()
+                    RETURNS TRIGGER AS
+                    $$
+                    BEGIN
+                       INSERT INTO "knowledge-elements_bigint"( id, source, status, "createdAt", "answerId", "assessmentId", "skillId", "earnedPix", "userId", "competenceId")
+                       VALUES ( NEW.id::BIGINT, NEW.source, NEW.status, NEW."createdAt", NEW."answerId", NEW."assessmentId", NEW."skillId", NEW."earnedPix", NEW."userId", NEW."competenceId");
+                      RETURN NEW;
+                    END
+                    $$ LANGUAGE plpgsql;`
+      )
+      .transacting(trx);
+
+    await knex
+      .raw(
+        `CREATE TRIGGER "trg_knowledge-elements"
+                    AFTER INSERT ON "knowledge-elements"
+                    FOR EACH ROW
+                    EXECUTE FUNCTION "insert_knowledge-elements_in_knowledge-elements_bigint"();`
+      )
+      .transacting(trx);
+  });
+  const endTime = performance.now();
+
+  logger.debug(`Call to copyNewlyInsertedRowsToAnswerIdTmpBigintTables took ${endTime - startTime} milliseconds`);
+  logger.debug('Finish adding triggers');
+};
+
+async function main() {
+  logger.debug('Start script... ');
+  await copyNewlyInsertedRowsToAnswerIdTmpBigintTables();
+  logger.debug('End script: done successfully.');
```

Comment:
Mettre au moins ce détail par défaut sinon on doit regarder le code retour pour savoir si c'est OK :sweat_smile: 
```suggestion
  logger.info('Start script... ');
  await copyNewlyInsertedRowsToAnswerIdTmpBigintTables();
  logger.info('End script: done successfully.');
```